src/mame/drivers/boogwing.cpp

```diff
@@ -1,5 +1,5 @@
 // license:BSD-3-Clause
-// copyright-holders:Bryan McPhail, David Haywood
+// copyright-holders:Bryan McPhail, David Haywood, cam900
```

Comment:
I don't think you should add yourself to the copyright holders list of the drivers where you only did minor changes.